mmv1/third_party/terraform/website/docs/r/bigtable_table.html.markdown

```diff
@@ -63,6 +71,9 @@ to delete/recreate the entire `google_bigtable_table` resource.
 * `project` - (Optional) The ID of the project in which the resource belongs. If it
     is not provided, the provider project is used.
 
+* `deletion_protection` - (Optional) A field to make the table protected against data loss i.e. when set to PROTECTED, deleting the table, the column families in the table, and the instance containing the table would be
+    prohibited. If not provided, currently deletion protection will be set to UNPROTECTED as it is the API default value.
```

Comment:
I think we just need to say that the default is UNPROTECTED. Let's not use word 'currently', because it sounds like it will change, which we don't know if it will, but it looks unlikely at this moment. We also don't have to justify it with the fact that it's API default.